README.md

```diff
@@ -101,6 +101,13 @@ See the [wiki](https://github.com/edenhill/librdkafka/wiki) for a FAQ.
 	zlib-dev (optional, for gzip compression support)
 	libssl-dev (optional, for SSL and SASL SCRAM support)
 	libsasl2-dev (optional, for SASL GSSAPI support)
+	libzstd-dev (optional, for ZStd compression support)
+
+**NOTE**: Static linking of ZStd enables encoding the original size in the
```

Comment:
Should the version be mentioned here since this actually only applies to zstd >=1.2.1